src/components/LoadingSpinner/LoadingSpinner.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import LoadingSpinner from './LoadingSpinner';
```

Comment:
This should be renamed to `LoadingSpinnerComponent` so that on line `13` you can export:

```
export const LoadingSpinner = ...
```